src/main/java/seedu/duke/common/Messages.java

```diff
@@ -131,6 +131,7 @@
     public static final String EXCEPTION_CATEGORY_NOT_FOUND = "~Error~ Invalid category.";
```

Comment:
for line 129, could you help to include "~Error~" at the front like the rest. thanks

Reply:
sure